test/mock/tr.go

```diff
@@ -0,0 +1,13 @@
+package mock
+
+type Tr struct {
+	TranslateF func(domain, metric string) string
```

Comment:
Strike the trailing `F` so mock field name = func name. (Other mocks in Blip should follow this convention.)

Reply:
It looks like the other mocks favor using `Func` rather than just `F`. Match that then?

Reply:
Correction (as you noted on Slack): "Func" suffix.